### ISSUE TITLE
Digest and turn threads into pdf

### ISSUE THREAD

**Issue by bbelderbos** [PyBites-Open-Source/karmabot#115]
There are some golden threads on the Pybites Slack, but sometimes you are too busy and don't necessarily want to bookmark and go back, how to make it easier to read them later?

Idea: tag a thread with an emoji that karmabot notices and then copies the thread out, generates a PDF and sends you in a DM.

**Comment by pogross** [comment]
Good idea. I don't know if you can send files directly via the bot API. Otherwise, we would need something like a S3 bucket and send a presigned URL let the user download it.

If you want to get fancy and have some spare money we could also add support for summarization of the thread via an LLM (e.g. OpenAI GPT3.5). 🤖 

Is there any preferred emoji (🔖) and limitations? Who can do this? Admin, anybody?

**Comment by bbelderbos** [comment]
Yep, we could use a private bucket + presigned URL.
Concern with GPT is privacy of user data.
Bookmark / 🔖 is nice yes.
Currently lower prio feature for me, but that's why these issues are useful.
